### ISSUE TITLE
Scheduler.step() missing, lr value

### ISSUE THREAD

**Issue by MarsJacobs** [EunhyeokPark/PROFIT#3]
Hi,
First of all, Thank you for this great Quantization method code.
I did several test with this code and checked several parameters.

And I've noticed that lr value has not changed in training.

Attached picture is lr graph in tensorboard.
<img width="705" alt="Screen Shot 2020-10-11 at 10 07 11 PM" src="https://user-images.githubusercontent.com/54992207/95679481-1e28d500-0c0e-11eb-9294-a345562d6e8e.png">

I added scheduler.step() and fixed this issue.
Could you check the lr value in this code?
Thank you.

**Comment by EunhyeokPark** [comment]
Thank you for letting me know.

There was a bug during refactoring my dirty lab code for upload.

I upload the fixed version.
